### ISSUE TITLE
Season grouping with separate Trackings

### ISSUE THREAD

**Issue by H1ghSyst3m** [aniyomiorg/aniyomi#1351]
### Describe your suggested feature

Most extension have multiple seasons in a single entry. The problem of this is, that the tracking with such entries are a little bit more difficult. 
They are mostly named with Season 1 Episode 1 or in German Staffel 1 Folge 1
Example Extension = Yomiroll

I have two suggestions
.
1)
Grouping the Seasons and separating the Tracking with the Seasons
So that we can, so that we can set for each season it's of Tracking
It would be also OK if it's not doing it automatically. It would be okay if we separate them manually. 
As example marking manually the episodes of Season 1 and with an option setting it as Season 1

2)
When we complete the season 1 (last episode of the season) and start a new season, that a Tracking dialogue pops up so that we can select a new tracking, since the Season 1 would then already automatically completed in the tracking.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/aniyomiorg/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.15.2.3](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Character95** [comment]
and can you add somewhere around anime title search category: Related anime Prequel, Sequel

**Comment by LittleFreak** [comment]
> ### Describe your suggested feature
> 
> Most extension have multiple seasons in a single entry. The problem of this is, that the tracking with such entries are a little bit more difficult. 
> They are mostly named with Season 1 Episode 1 or in German Staffel 1 Folge 1
> Example Extension = Yomiroll
> 
> I have two suggestions
> 
> 1)
> Grouping the Seasons and separating the Tracking with the Seasons
> So that we can, so that we can set for each season it's of Tracking
> It would be also OK if it's not doing it automatically. It would be okay if we separate them manually. 
> As example marking manually the episodes of Season 1 and with an option setting it as Season 1
> 
> 2)
> When we complete the season 1 (last episode of the season) and start a new season, that a Tracking dialogue pops up so that we can select a new tracking, since the Season 1 would then already automatically completed in the tracking.

Grouping would be nice to have.
But I don't see your point with tracking, especially if you use sources, you which are already grouping seasons and specials together (like yomiroll, aniworld...)
Also seasons aren't always declared right. Yomiroll counts a 2nd cour still as the same season, sometimes starts with season 2 and forward.
So we are already have to manage the tracking manually midway - without any annoying popup.

**Comment by AGA2707** [comment]
Yes I also want to suggest this. For example on Yomiroll. All seasons and movies are all in one entry and you can only add one tracker so tracking all seasons and movies are impossible.

**Comment by LittleFreak** [comment]
> Yes I also want to suggest this. For example on Yomiroll. All seasons and movies are all in one entry and you can only add one tracker so tracking all seasons and movies are impossible.

I guess there is no need to implement the tracker restriction, since you select the anime to track anyway.

**Comment by AGA2707** [comment]
> > Yes I also want to suggest this. For example on Yomiroll. All seasons and movies are all in one entry and you can only add one tracker so tracking all seasons and movies are impossible.
> 
> I guess there is no need to implement the tracker restriction, since you select the anime to track anyway. 

Tracker don't know the seasons. And only recognize the number on episodes, what if the entry have 2 seasons and every seasons have the same number of episode, tracker can't recognize it. You need to adjust it manually.

**Comment by LittleFreak** [comment]
> So the aniyomi can detect the S and E and group them to separate the seasons in the entry. And then when I start the next season Tracking can ask for the new tracker
You mean as semi automatic popup? That could be possible for sure, but 2nd cours would still contradict this feature.

**Comment by AGA2707** [comment]
> > > > Yes I also want to suggest this. For example on Yomiroll. All seasons and movies are all in one entry and you can only add one tracker so tracking all seasons and movies are impossible.
> > > 
> > > 
> > > I guess there is no need to implement the tracker restriction, since you select the anime to track anyway.
> > 
> > Tracker don't know the seasons. And only recognize the number on episodes, what if the entry have 2 seasons and every seasons have the same number of episode, tracker can't recognize it. You need to adjust it manually.
> 
> And this won't likely change. Take Yomiroll for example... There are many entries, where the 2nd cour of a season is just counting on, suggesting it's the same season, while MAL needs a separate entry...
> OR
> I've also seen Seasons in Yomirolz, which suddenly increasing to S34 or anything hilarious like that.
> Thats the reason, why you'll need to add/switch the tracker manually anyway. 

I give you a reference for this we will use Jujutsu Kaisen for example from Crunchyroll it has 2 seasons and one movie in one entry and giving it a tracker is useless cause it can't track s1 e1 or s2 e1 or movie e1 for example, if you want you need to adjust the tracker manually and after you watch one season you will change the tracker and adjust it manually after you watch. Sorry for my bad English I hope you understand it.![Screenshot_2024-02-03-17-57-09-794_org.mozilla.firefox.jpg](https://github.com/aniyomiorg/aniyomi/assets/148602118/155056f7-710c-48e3-92c8-eab6fef82eb9)

**Comment by LittleFreak** [comment]
> > So the aniyomi can detect the S and E and group them to separate the seasons in the entry. And then when I start the next season Tracking can ask for the new tracker
> 

Sorry, the following answer of mine was caught up in the quote. I guess it would describe a useful approach to your idea? 
> You mean as semi automatic popup? That could be possible for sure, but 2nd cours would still contradict this feature.
